crates/walrus-service/src/node.rs

```diff
@@ -1178,6 +1178,9 @@ impl StorageNode {
                 // TODO: Implement DenyListEvent handling (WAL-424)
                 event_handle.mark_as_complete();
             }
+            EventStreamElement::ContractEvent(ContractEvent::ProtocolEvent(_protocol_event)) => {
+                todo!("Handle protocol events");
```

Comment:
Suggestion: we should not panic with a TODO here. Any node running this version of the software is not expecting a protocol version update, so we should panic explicitly, something like this:

```suggestion
                panic!("unexpected protocol version update: {_protocol_version}");
```